src/wasm.rs

```diff
@@ -249,3 +249,28 @@ pub async fn read_parquet_stream(
     });
     Ok(wasm_streams::ReadableStream::from_stream(stream).into_raw())
 }
+
+#[wasm_bindgen(js_name = "transformParquetStream")]
```

Comment:
Can you add a docstring here? Potentially include an example too? (The docstring will be included in the typescript-generated typedefs and seen by JS users)

Reply:
Maybe include an example of how you can pass in a `File` handle? So you can write out to a Parquet file on disk without materializing the buffer in memory?